### ISSUE TITLE
Kraken good accessibility - Completed 

### ISSUE THREAD

**Issue by samanthadasilva** [samanthadasilva/kraken-good-accessibility#1]
@thatkidmadrid

**Comment by thatkidmadrid** [comment]
Nice!
I think the roles you set can make sense, but the way I had it was the header with the title and author as the `role="banner"`.
for the image and pull quote you have them set up as just sections, but for the screen reader, it would be more useful to set them up as a `figure` with a `figcaption` This will help tie the two elements together

Also, aria-labels for the links should be thought of as what the screenreader should read, and not so much describing the image like an alt attribute. For example you have for one of the links with an icon:

```
aria-label="icon labels for links"

or

aria-label="icon for navigation"
```
To me this seems to be describing the icon and not the link. You also have this repeating for all of the links which means they lose their usefulness to the screen reader - the screen reader will read out the same text for each link which does different things...

Instead I would write something like "Go to the previous chapter"

<img width="497" alt="Screen Shot 2021-10-03 at 5 17 28 PM" src="https://user-images.githubusercontent.com/6340290/135771869-b4eda9d9-edfa-48e1-aa6d-3cf03b88fbcd.png">
<img width="500" alt="Screen Shot 2021-10-03 at 5 17 50 PM" src="https://user-images.githubusercontent.com/6340290/135771870-f62f0285-bf35-4348-b4fe-d0e85a1afb54.png">

see the difference in the rotors for how the links appear to the screen reader

Make these changes and I will add the grade to LTW

**Comment by samanthadasilva** [comment]
@thatkidmadrid 
I made the changes!

**Comment by thatkidmadrid** [comment]
Added the grade to LTW
